loopchain/peer/block_manager.py

```diff
@@ -397,7 +397,7 @@ def __block_request_by_citizen(self, block_height, rs_rest_stub):
         block_version = self.get_blockchain().block_versioner.get_version(block_height)
         block_serializer = BlockSerializer.new(block_version, self.get_blockchain().tx_versioner)
         block = block_serializer.deserialize(get_block_result['block'])
-        confirm_info = get_block_result['confirm_info'] if 'confirm_info' in get_block_result else None
+        confirm_info = get_block_result.get('confirm_info', '').encode('utf-8')
```

Comment:
I think `confirm_info` already converted by `encode('utf-8')` your [PR](https://github.com/icon-project/icon-rpc-server/pull/61/files#diff-9761d03b1583578cd176e1cb0f2e5759R157) in icon-rpc-server.

Reply:
my misunderstanding